src/containers/App/index.js

```diff
@@ -55,6 +55,9 @@ export const AppContainer = () => {
   const isFrontOffice = useIsFrontOffice();
   const headerIsTall = isFrontOffice && !isAuthenticated();
 
+  useInjectSaga({ key: 'global', saga });
+  useInjectReducer({ key: 'global', reducer });
```

Comment:
Sinds this is a global model definition now, woudn't be beter to place this in the `src/models/index.js` where the rest of global models are injected?

Reply:
I'm not sure what you mean. The `categories` model is already instantiated in `app.js`. The `App` container has its own reducer and saga that need to be injected.

Reply:
What I mean is that the `loadModules` function that is defined in `src/models/index.js` is responsible for loading the redux models and it will be good to have the `global` model at this place loaded for consistency.